mne_features/univariate.py

```diff
@@ -547,7 +557,27 @@ def compute_pow_freq_bands(sfreq, data, freq_bands=np.array([0.5, 4., 8., 13.,
     if normalize:
         pow_freq_bands = np.divide(pow_freq_bands,
                                    np.sum(ps, axis=-1)[:, None])
-    return pow_freq_bands.ravel()
+    if ratios is None:
+        return pow_freq_bands.ravel()
+    elif ratios not in ['all', 'only']:
+        raise ValueError('The given value (%s) for the parameter `ratios` '
+                         'is not valid. Valid values are: `all` or `only`.'
+                         % str(ratios))
+    elif ratios is None:
+        return pow_freq_bands.ravel()
+    else:
+        n_freq_bands = n_freqs - 1
+        band_ratios = np.empty((n_channels, n_freq_bands * (n_freq_bands - 1)))
+        pos = 0
+        for idx in np.ndindex((n_freq_bands, n_freq_bands)):
+            if idx[0] != idx[1]:
+                band_ratios[:, pos] = np.divide(pow_freq_bands[:, idx[0]],
+                                                pow_freq_bands[:, idx[1]])
+                pos += 1
```

Comment:
```
for pos, (i, j) in enumerate(np.ndindex((n_freq_bands, n_freq_bands))):
      if i != j:
              band_ratios[:, pos] = pow_freq_bands[:, i] / pow_freq_bands[:, j]
```

Reply:
I thought about that but we need to enumerate all the off-diagonal coefficients.
Here, if `i == j`, `pos` is incremented and it will eventually take larger values than `band_ratios.shape[-1]`.

Reply:
One solution could be to change `triu_idx` in `mne_features/utils.py` into something like:
```
def idxiter(n, triu=True, include_diag=False):
    pos = -1
    for i in range(n):
        for j in range(i * int(triu), n):
            if not include_diag and i == j:
                continue
            else:
                pos += 1
                yield pos, i, j
```
This way, `idxiter(n, triu=True)` allows to iterate over the (strict) upper triangular part of a square array and `idxiter(n, triu=False)` allows to iterate over all non-diagonal entries.